how-to-guide.org

```diff
@@ -128,6 +128,42 @@ add_subdirectory("python")
 #+END_SRC
 Now, the python bindings will be built alongside your project.
 ** General
+*** Contribute to someone else's project
+Having a common software repository helps us to share and use code from other members, but also contribute to them. This implies obtaining the source code (and not the compiled binaries or the conda package) from Github and commit the changes. Obviously, commiting changes comports a risk of introducing bug or unwanted features for the software. You should therefore commit your changes in a separate branch and open a merge request. After inspection, your changes will eventually be accepted and incorporated into the main branch. 
+Let us now describe this procedure step by step.
+
+First, checkout the source code and cd to this directory.
+#+BEGIN_SRC tmux
+git pull git@gitlab.com:tomo-ml/msd_pytorch.git
+cd ~/projects/msd_pytorch/
+#+END_SRC
+
+Change something:
+#+BEGIN_SRC tmux
+touch atoha
+#+END_SRC
+
+Stage file:
+#+BEGIN_SRC tmux
+git add atoha
+#+END_SRC
+
+#+BEGIN_SRC tmux
+git checkout -b henribranch
+#+END_SRC
+
+#+BEGIN_SRC
+git commit -m "For Henri"
+#+END_SRC
+
+#+BEGIN_SRC tmux
+git push origin henribranch
+#+END_SRC
+
+Open the link that is displayed, in this case
+https://gitlab.com/tomo-ml/msd_pytorch/merge_requests/new/henribranch.
+Describe your changes, then finally click submit merge request.
```

Comment:
The text Allard wrote for the `attic` repository may be more appropriate here:

> You can add code to this repository using a pull request. To make a pull request:
> 
> Fork this repository (Look for the forky railway track icon near the top right of the page)
> Clone your fork to your local filesystem
> Make your changes
> Commit your changes
> Push your changes to your forked repository
> Make a pull request (a button to do this should show up on the github page of your forked repository)
> For more information about pull requests, see https://help.github.com/articles/about-pull-requests/

Reply:
Hm, does it mean that we encourage fork over branches ?

Reply:
I would prefer forks over branches, to avoid having a lot of (inactive) branches in repositories. Either that, or we have add instructions on deleting (remote) branches.

Reply:
I agree that usually doing a PR from a personal fork is preferred over a PR from a branch in the main repository.

Another thing is that I would make the example more feature-oriented than person-oriented. E.g., a commit "Reverse the polarity of the neutron flow" on a branch "reverse-polarity", instead of the current "For Henri".